src/components/RightSidebar/SharedItems/SharedItems.vue

```diff
@@ -74,51 +52,17 @@ export default {
 			type: Object,
 			required: true,
 		},
+
+		// Limits the amount of items displayed
+		limit: {
+			type: Boolean,
```

Comment:
should inject the limit instead of hardcoding it?

Reply:
yes, I'm going to need this for deck cards and locations which will be prob 3 only. So next pr